src/kirby/components/chip/chip.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, Input, HostBinding } from '@angular/core';
+
+@Component({
+  selector: 'kirby-chip',
+  templateUrl: './chip.component.html',
+  styleUrls: ['./chip.component.scss'],
+})
+export class ChipComponent {
+  @Input() text: string;
+  @Input()
+  @HostBinding('class.isSelected')
```

Comment:
Use css naming convention:
```suggestion
  @HostBinding('class.is-selected')
```

Reply:
Done